### PR TITLE
CNI calico-ipam version flag

### LINKED ISSUE #195: Calico IPAM plugin not reporting version
After installing calico/calico-ipam 1.4.2.  Whenever I call `/opt/cni/bin/calico-ipam -v` I receive this message `CNI_COMMAND env variable missing
{
    "code": 100,
    "msg": "required env variables missing"
}`

### PR DESCRIPTION
**Pull request by gunjan5**
Fixes #195

### CODE REVIEW

**Comment by robbrockbank** [comment]
LGTM

Is it possible to test this?

**Comment by gunjan5** [comment]
already built it and tested it and seems to work
```
$ ./calico-ipam -v
v1.5.1-8-g3aa3c0f-dirty
```

**Comment by robbrockbank** [comment]
I meant with automated test?

**Comment by gunjan5** [comment]
No, we don't have any for testing the version, don't think it's super critical

File: ipam/calico-ipam.go [hunk 1]
```diff
@@ -29,7 +30,32 @@ import (
...
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	if *version {
+		fmt.Println(VERSION)
+		os.Exit(0)
+	}
+
+	if err := utils.AddIgnoreUnknownArgs(); err != nil {
```
Comment (gunjan5) [inline ipam/calico-ipam.go]: yup, I misunderstood what it does.

**Pull request merged**